packages/@react-spectrum/tabs/src/Tabs.tsx

```diff
@@ -319,14 +334,10 @@ export function TabPanels<T>(props: SpectrumTabPanelsProps<T>) {
     tabPanelProps['aria-labelledby'] = ctxTabPanelProps['aria-labelledby'];
   }
 
-  const factory = nodes => new ListCollection(nodes);
-  const collection = useCollection({items: tabProps.items, ...props}, factory, {suppressTextValueWarning: true});
-  const selectedItem = tabListState ? collection.getItem(tabListState.selectedKey) : null;
-
   return (
     <FocusRing focusRingClass={classNames(styles, 'focus-ring')}>
-      <div {...styleProps} {...tabPanelProps} ref={ref} key={tabPanelProps.id} className={classNames(styles, 'spectrum-TabsPanel-tabpanel')}>
-        {selectedItem && selectedItem.props.children}
+      <div {...styleProps} {...tabPanelProps} ref={ref} className={classNames(styles, 'spectrum-TabsPanel-tabpanel')}>
```

Comment:
So do we need to split this all up into two? or could we just make a one line change and set the key on here to be `tabListState?.selectedKey`?

Reply:
I was trying to get the key as a one line change to work. When Devon and I looked into it we realized that making the distinction of TabPanels with a TabPanel child solves the problem. The unique and separate TabPanel children works better with the mutation observer.